.eleventy.js

```diff
@@ -32,6 +34,32 @@ module.exports = (config) => {
     'node_modules/@moduk/frontend/dist/client/moduk-frontend.umd.js': 'moduk-frontend.umd.js',
   })
 
+  config.addTemplateFormats('ts')
+  config.addExtension('ts', {
+    read: false,
+    permalink: false,
+    outputFileExtension: 'js',
+    compile: async (_inputContent, filename) => {
+      const compiler = webpack({
+        ...webpackConfig,
+        entry: filename,
+      })
+
+      return () =>
+        new Promise((resolve, reject) => {
+          compiler.run((err) => {
+            compiler.close((closeErr) => {
+              if (err || closeErr) {
+                reject(err)
+              } else {
+                resolve()
+              }
+            })
+          })
+        })
+    },
+  })
+
```

Comment:
I wanted to leverage 11ty ecosystem with webpack also watches the `.ts` files. I did some research a lot of the methods use `npm-run-all` however that has not had an update in 4 years.